cudax/CMakeLists.txt

```diff
@@ -20,10 +20,15 @@ enable_language(CUDA)
 option(cudax_ENABLE_HEADER_TESTING "Test that CUDA Experimental's public headers compile." ON)
 option(cudax_ENABLE_TESTING "Build CUDA Experimental's tests." ON)
 option(cudax_ENABLE_EXAMPLES "Build CUDA Experimental's examples." ON)
+option(cudax_ENABLE_CUDASTF_CODE_GENERATION "Enable code generation using STF's parallel_for or launch with CUDA compiler." ON)
```

Comment:
This should match what we have in the preset, so I believe this should either here be `OFF` or the preset should mark the as `true`

Reply:
I'm going to change the preset to true, i was currently trying to ensure it compiles fine with false as an intermediate state